packages/core/src/Account.ts

```diff
@@ -301,7 +297,7 @@ export class Account<T = any> extends EventEmitter {
     await this.apiClient.transport.http.associateClientWithSession(validClient.id);
 
     await this.service.proteus.initClient(this.storeEngine, this.apiClient.context);
-    if (this.backendFeatures.supportsMLS && this.cryptoProtocolConfig?.mls) {
+    if (this.service.mls) {
```

Comment:
from now on, we know that if the mls service has been instantiated we are able to use MLS everywhere. No need to check for the config or the backend abilities anymore